packages/ajax/src/cache.js

```diff
@@ -147,7 +146,7 @@ export const searchParamSerializer = (params = {}) =>
  * and proactively clean it
  * @param {string} cacheIdentifier usually the refreshToken of the owner of the cache
  */
-const getCache = cacheIdentifier => {
+export const getCache = cacheIdentifier => {
```

Comment:
If we export this now, we should have some tests for it I think that test this import and its functionality in isolation